test/parallel/test-util-inspecttag.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const util = require('util');
+
+const inspectTag = util.inspectTag;
+
+const obj = {a: 1};
+
+assert.strictEqual(`${obj}`, '[object Object]');
+assert.strictEqual(inspectTag`${obj}`, '{ a: 1 }');
+assert.strictEqual(inspectTag`${obj}-${obj}`, '{ a: 1 }-{ a: 1 }');
```

Comment:
Worth add a test when the end is a string(`n === keys.length`), and a test when there are no variables in the template.